src/main/java/org/folio/des/scheduling/bursar/BursarScheduledJobInitializer.java

```diff
@@ -18,11 +22,15 @@ public class BursarScheduledJobInitializer {
   private final ExportConfigService burSarExportConfigService;
   private final BursarExportScheduler bursarExportScheduler;
 
-  public void initAllScheduledJob() {
+
+  private final SemVer quartzEdifactMinVersion = new SemVer("3.0.0-SNAPSHOT");
```

Comment:
can be changed to quartzBursarMinVersion/quartzMinVersion, it's not about edifact